src/main/webapp/app/overview/course-exercises/course-exercises.component.html

```diff
@@ -1,21 +1,20 @@
 <div class="row">
     <div class="col-12 col-md-8">
         <div class="mt-3 position-relative" *ngIf="course">
-            <div class="exercise-controls d-none d-sm-block" *ngIf="course?.exercises.length > 0">
-                <div ngbDropdown placement="bottom-right" class="d-inline-block">
-                    <button class="btn btn-outline-primary" id="dropdownBasic1" ngbDropdownToggle>
-                        {{ 'artemisApp.courseOverview.exerciseList.sortExercises' | translate }}
+            <div class="exercise-controls flex d-sm-block" *ngIf="course?.exercises.length > 0">
+                <div class="d-flex align-items-center mr-2">
+                    <label class="d-flex align-items-center mb-0 mr-2">
+                        <input type="checkbox" [checked]="activeFilters.has(filterType.OVERDUE)" (change)="filterUpdate([filterType.OVERDUE])" />
+                        <span class="ml-2" jhiTranslate="artemisApp.courseOverview.exerciseList.overdue">Hide overdue</span>
+                    </label>
+                    <label class="d-flex align-items-center mb-0 mr-2">
+                        <input type="checkbox" [checked]="activeFilters.has(filterType.NEEDS_WORK)" (change)="filterUpdate([filterType.NEEDS_WORK])" />
+                        <span class="ml-2" jhiTranslate="artemisApp.courseOverview.exerciseList.fullScore">Hide overdue</span>
```

Comment:
I think you missed this one, should be `Hide full score`

Reply:
True!